sections/v.zsh

```diff
@@ -0,0 +1,41 @@
+#
+# V
+#
+# V is a languaje program designed to be "Simple, fast, safe, compiled. For developing maintainable software."
```

Comment:
typo `languaje`

Suggestion: V is a simple, fast, safe, compiled language for developing maintainable software.